Obsolete_Deprecated/IORefCAS/IORefCAS.cabal

```diff
@@ -6,7 +6,8 @@ Author:              Adam C. Foltzer, Ryan Newton
 Maintainer:          acfoltzer@gmail.com, rrnewton@gmail.com
 Category:            Data
 Build-type:          Simple
-Cabal-version:       >=1.8
+Cabal-version:       >=1.18
```

Comment:
Is there a reason that `Cabal-version` needs to be increased?

Reply:
I vaguely remembered 1.18 is needed for using some stanza such as `Simple` build type, as well as specifying `default-languages` and `default-extensions`, some packages such as (cryotonite)[https://github.com/haskell-crypto/cryptonite/blob/4622e5fc8ece82f4cf31358e31cd02cf020e558e/cryptonite.cabal#L38] uses 1.18 as well